src/packets.toit

```diff
@@ -93,27 +85,128 @@ abstract class Packet:
     buffer.write length
     buffer.write str
 
+  static encode_uint16 value/int -> ByteArray:
+    result := ByteArray 2
+    binary.BIG_ENDIAN.put_uint16 result 0 value
+    return result
+
+  static decode_string reader/reader.BufferedReader -> string:
+    length := decode_uint16 reader
+    return reader.read_string length
+
+  static decode_uint16 reader/reader.BufferedReader -> int:
+    length_bytes := reader.read_bytes 2
+    return binary.BIG_ENDIAN.uint16 length_bytes 0
+
+  static debug_string_ packet/Packet -> string:
```

Comment:
This might not be worth keeping around for users of the mqtt package. Not entirely sure what the best way to handle that is.

Reply:
It's a private method.
I will remove the `stringify` below (which I had only enabled for debugging), in which case it should only be used by this package.

I use it a lot in the logging messages of the broker.
Can be convinced to remove it entirely.